### ISSUE TITLE
Support PH divinations based on year

### ISSUE THREAD

**Issue by HolgerJeromin** [opening-hours/opening_hours.js#230]
2017-10-31 is special as this is PH in Germany.
https://de.wikipedia.org/wiki/Reformationstag#Deutschland

Is this covered in this library?

**Comment by ypid** [comment]
Nope. I already looked into this but PH for one year only can not be expressed so I went with ignoring it. Not ideal but for now I would say this is the practical approach. If this happens more (I am sure it will, I guess we will have to implement it).

**Comment by KristjanESPERANTO** [comment]
Implemented via #628. Will be in the net release :)

**Comment by HolgerJeromin** [comment]
Oh. Wow. 
I completely have forgotten this issue :)

**Comment by KristjanESPERANTO** [comment]
This has been fixed in release v3.15.0 🙂
